product_profile/product.py

```diff
@@ -79,52 +79,51 @@ def _get_profile_fields(self):
                 if field not in fields_to_exclude]
 
     @api.model
-    def _get_profile_data(self, profile_id, filled_fields=None):
+    def _get_vals_from_profile(self, profile_id):
```

Comment:
I propose to maintain the old method which call the new method and write instruction for future refactoring in v9. Then no api changed but future v9 migration becomes clearly defined

What do you think ?